RecoMET/METProducers/python/METSignificanceParams_cfi.py

```diff
@@ -44,5 +44,6 @@
 
 from Configuration.ProcessModifiers.run2_miniAOD_UL_cff import run2_miniAOD_UL
 from Configuration.Eras.Modifier_run2_nanoAOD_106Xv1_cff import run2_nanoAOD_106Xv1
-(run2_miniAOD_UL|run2_nanoAOD_106Xv1).toModify(METSignificanceParams, useDeltaRforFootprint = True)
-(run2_miniAOD_UL|run2_nanoAOD_106Xv1).toModify(METSignificanceParams_Data, useDeltaRforFootprint = True)
+from Configuration.Eras.Modifier_run2_nanoAOD_106Xv2_cff import run2_nanoAOD_106Xv2
+(run2_miniAOD_UL|run2_nanoAOD_106Xv1|run2_nanoAOD_106Xv2).toModify(METSignificanceParams, useDeltaRforFootprint = True)
+(run2_miniAOD_UL|run2_nanoAOD_106Xv1|run2_nanoAOD_106Xv2).toModify(METSignificanceParams_Data, useDeltaRforFootprint = True)
```

Comment:
is any of this needed in the master branch?

Reply:
No I do not think they are needed in the master. @lathomas @kirschen

Reply:
This are already the settings in the master branch.
We backported back then but pipelined only for the nanov8 (run2_nanoAOD_106Xv1), but now we had to set explicitly for the nanov9 run2_nanoAOD_106Xv2 as well. Ideally we should set this as default and then reset for the nanos before UL (while keeping not the default in mini)